### DISCUSSION TITLE
Not able to see the Video

### DISCUSSION THREAD

**Discussion by Manju9876** [learndevopsonline/learndevopsonline#743] [Q&A] [answered]
hello, @r-devops sir
I'm not able to see the video of 27th April.

###D72-2023-04-27-SESSION-26 

###I guess my mail Id has no  access

###could you please let me know what happened whether you  uploaded or not, because I didn't attended the morning class of 27th April.

**Comment by Nagamanidevops** [comment]
I think there was no session on that day

**Reply by Manju9876** [reply]
hi @Nagamanidevops 
I'm talking about 27th April, 
are you saying that there was no class on 27th of April morning?

**Reply by Arafique458** [reply, marked as answer]
Hi @Manju9876 

There was no class, I believe Raghu Sir sent email, regarding that he wasn't able to join the session on 27th April.

I hope this helps.

**Reply by Manju9876** [reply]
hi @Arafique458 
###I got it, Thanks very much.
